ui/views_test.py

```diff
@@ -374,26 +374,6 @@ def test_program_thumbnail(self):
         resp = self.client.get('/')
         self.assertContains(resp, image.get_rendition('fill-690x530').url)
 
-    def test_courses_enrollment_text(self):
```

Comment:
No, I dropped this test. It looks like it's asserting that the `get_course_enrollment_text()` function is called once for every course object. However, now that this function has been rolled into the `Course` model, it doesn't really make sense to assert on that. Maybe we could accomplish the same thing by writing a test to ensure that every Course model's enrollment text is present on the page -- but that seems like a very specific, rather brittle thing to test on. Do you think that's a good idea? Anything else I should test instead?